manager/encryption/fernet.go

```diff
@@ -0,0 +1,54 @@
+package encryption
+
+import (
+	"fmt"
+
+	"github.com/docker/swarmkit/api"
+
+	"github.com/fernet/fernet-go"
+)
+
+// Fernet wraps the `fernet` library as an implementation of encrypter/decrypter.
+type Fernet struct {
+	key fernet.Key
+}
+
+// NewFernet returns a new Fernet encrypter/decrypter with the given key
+func NewFernet(key []byte) Fernet {
+	frnt := Fernet{}
+	copy(frnt.key[:], key)
+	return frnt
+}
+
+// Algorithm returns the type of algorithm this is (NACL Secretbox using XSalsa20 and Poly1305)
```

Comment:
Comment seems to be a carryover from the non-fips algo.

Reply:
Yes, thanks for catching that!